src/modules/Elsa.MassTransit/Options/MassTransitWorkflowDispatcherOptions.cs

```diff
@@ -5,6 +5,8 @@ namespace Elsa.MassTransit.Options;
 /// Provides options to the <see cref="DispatchWorkflowRequestConsumerDefinition"/>
 public class MassTransitWorkflowDispatcherOptions
 {
+    /// The TTL of queues that are seen as short lived (typically queues that are created per running instance).
+    public TimeSpan? ShortTermQueueLifetime { get; set; }
```

Comment:
Some alternative (subjectively, better) options:

- `TemporaryQueueTtl`
- `TempQueueTtl`
- `TemporaryQueueLifetime`
- `TempQueueLifetime`

Reply:
Changed it to TemporaryQueueTtl